app/(home)/nisp/page.tsx

```diff
@@ -1,10 +1,22 @@
 import PageHero from "@/components/shared/PageHero";
+import Link from "next/link";
 import React from "react";
 
 const Nisp = () => {
   return (
-    <section>
+    <section className="p-3">
       <PageHero pageHeroTitle="NISP" />
+      <div className="max-w-[1500px] mx-auto flex flex-col md:flex-row justify-center items-center gap-8 md:gap-[10%]">
+        <div className="text-justify">
+          <h2 className="text-center font-semibold text-2xl my-5">WHAT IS NISP ?</h2>
```

Comment:
give this the same style of default header with green which is used in other pages

A better way would be to create a component in components/shared  for the title and use the same across all the necessary places